interface/themes/style_light.scss

```diff
@@ -308,6 +308,7 @@ div.category-display {
 .chevron_color {
     color: #2672ec;
 }
+
```

Comment:
you also need it in the style_light.scss script

Reply:
done . I'm pretty confused about this new styling mechanism - what goes where